gapis/api/cmd_id_group.go

```diff
@@ -33,9 +33,10 @@ import (
 // or an command range that is within this group's span but outside of any
 // sub-group.
 type CmdIDGroup struct {
-	Name  string     // Name of this group.
-	Range CmdIDRange // The range of commands this group (and items) represents.
-	Spans Spans      // All sub-groups and sub-ranges of this group.
+	Name      string     // Name of this group.
+	Range     CmdIDRange // The range of commands this group (and items) represents.
+	Spans     Spans      // All sub-groups and sub-ranges of this group.
+	Thumbnail CmdID      // The command which shall be used to make thumbnail for this group
```

Comment:
I really don't think this information belongs here. The types in the `gapis/api` package are supposed to be core and generic data structures relating to APIs, Commands and basic resource types. Thumbnail is a very specific usage.

It's unfortunate that the `resolve.commandTree` structure doesn't have a separate node type for its representation of the command tree. 

Given that you're just doing a binary search on the events list to get the thumbnail for the group, you could just do this on demand in [`resolve.CommandTreeNode()`](https://github.com/google/gapid/blob/f0fccfd6c035b4feff3124ed19dffb6b251de2b4/gapis/resolve/command_tree.go#L78-L119)